src/ex4/db/migrations/20220621101713-create-item.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        allowNull: false,
+        primaryKey: true,
+      },
+      ItemName: {
```

Comment:
```suggestion
      item_name: {
```

please change all columns accordingly. don't forget to drop the database and run the migrations again in order for it to take effect.

The should still be camelCase though, so in this case you can use `underscored: true` in the `tableOptions` given to sequelize, so you will still be able to access `item.itemName` instead of `item.item_name` in the code

Reply:
consider: btw it's better to have concise column name like `name` - because it's obvious that this name is related to item - because it's under the `items` table!